libraries/JGUZDV.Blazor.Components/src/DarkModeSelector.razor

```diff
@@ -0,0 +1,21 @@
+﻿@inject IStringLocalizer<ComponentStrings> LR
+
+
+<Dropdown DropDownMenuClass="dropdown-menu-end">
+    <DropdownLabelContent>
+        <span class="btn btn-none" id="jgu-current-theme-icon" aria-haspopup="true" aria-expanded="false" aria-label="@LR["DarkModeSelector.Header"]">
+            <i class="fas fa-adjust"></i>
+        </span>
+    </DropdownLabelContent>
+    <DropdownContent>
+        <button class="dropdown-item" id="jgu-light-theme-icon" onclick="JGUZDVBlazorComponents.applyTheme('light', false)">
+            @Icons.LightMode("small pe-1") @LR["Theme.Light"]
+        </button>
+        <button class="dropdown-item" id="jgu-dark-theme-icon" onclick="JGUZDVBlazorComponents.applyTheme('dark', false)">
```

Comment:
Wir hatten hier noch darüber gesprochen, das nicht per onClick sondern data-set-theme="dark" abzubilden.
Das kannst du dann in der lib.js per document.ready(() => document.querySelectorAll(..)) auf onClick registrieren. Das ist wichtig, falls es CSP gibt, die inline JS verbieten.